src/core/qgsproject.cpp

```diff
@@ -900,6 +912,24 @@ static void _getTitle( const QDomDocument &doc, QString &title )
 
 }
 
+static void getProjectMetadata(const QDomDocument &doc, QString &lastUser, QString &lastUserFull)

```

Comment:
Wouldn't be better named getProjectUserInformation ? We actually do have other metadata in the project file.

Reply:
Sorry, forgot to mention: QT and QGIS policy is to not use `get` for getters, just `projectUserInformation()` will do.

Reply:
ok. good to know